Formula/m/manim.rb

```diff
@@ -115,8 +93,8 @@ class Manim < Formula
   end
 
   resource "moderngl-window" do
-    url "https://files.pythonhosted.org/packages/c2/90/14a7e798c9310dfa5ee94716d0b73f465780741ae6c9d45eb21626d87bf6/moderngl-window-2.4.4.tar.gz"
-    sha256 "66a9c5412b5eb0a5ca7cda351e4484ce02a167cf87eb4dc59bb82439c58130b5"
+    url "https://files.pythonhosted.org/packages/4e/d7/3467a5cb0cbd3b9bb15c6a10f793f59bb606862c1c7c704af483156e5c7c/moderngl-window-2.4.5.tar.gz"
+    sha256 "2ee6e80b215ce4e7db5bef8d166010a3f2b9060449688652bfe06fa7c904d01b"
   end
```

Comment:
did you pin the version to 2.4.5? I just raised an issue for this. :) 

https://github.com/moderngl/moderngl/issues/691

Reply:
yup, you beat me to it!